src/ArgParse.jl

```diff
@@ -1,10 +1,9 @@
-isdefined(:OptionsMod) || require("Options")
+isdefined(:OptionsMod) || @eval import Options
```

Comment:
I don't think `@eval` is needed here, is it?
Also, I'm not on latest julia master: does this still print a warning?